synapse/handlers/message.py

```diff
@@ -260,7 +260,6 @@ def maybe_schedule_expiry(self, event):
         Args:
             event (EventBase): The event to schedule the expiry of.
         """
-        assert not self._is_worker_app
```

Comment:
Looks like `self._is_worker_app` is no longer used now.

Reply:
good shout